applications/machine_learning/configuration/thingy53_nrf5340_cpuapp_ns/app_ZRelease.conf

```diff
@@ -60,6 +62,8 @@ CONFIG_EI_WRAPPER_DATA_BUF_SIZE=1000
 ################################################################################
 # System configuration
 
+CONFIG_PM_POLICY_APP=y
```

Comment:
does power manager depends on this option (or maybe it should select it?)

Reply:
seen your discussion at the top already :)

Reply:
The dependency is here already:
```
onfig CAF_POWER_MANAGER
	bool "Enable power management"
	select PM_DEVICE
	select CAF_PM_EVENTS
	select CAF_POWER_MANAGER_EVENTS
	depends on PM_POLICY_APP
	help
	  Enable power management, which will put the device to low-power mode
	  if it is idle.
```

Reply:
It is first commit in this PR